_posts/2013-04-06-developing-addons-and-blueprints.md

```diff
@@ -138,6 +138,20 @@ Let's add some meta data to categorize the addon a little better:
 ]
 {% endhighlight %}
 
+### Addons depending on other addons
+If your addon depends on another addon, install it as a dependency in your
+`package.json`:
+
+{% highlight javascript %}
+// ...
+"dependencies": {
+  "ember-ajax": "^0.7.1"
+}
+{% endhighlight %}
+
+An application that consumes your addon will automatically install
+and bundle dependencies specified this way.
```

Comment:
@knownasilya That seems to be how it works. It seems to be added to `vendor.js`:

```bash
# From the parent app:
> cat dist/assets/vendor.js|grep ember-ajax
;define('ember-ajax/errors', ['exports', 'ember'], function (exports, Ember) {
define('ember-ajax/index', ['exports', 'ember-ajax/request'], function (exports, request) {
define('ember-ajax/make-promise', ['exports', 'ember'], function (exports, Ember) {
    }, 'ember-ajax: ' + type + ' to ' + settings.url);
define('ember-ajax/raw', ['exports', 'ember-ajax/make-promise', 'ember-ajax/utils/parse-args', 'ember'], function (exports, makePromise, parseArgs, Ember) {
    deprecate('ember-ajax/raw is deprecated and will be removed in ember-ajax@2.0.0', false, { id: 'ember-ajax.raw' });
define('ember-ajax/request', ['exports', 'ember-ajax/raw', 'ember'], function (exports, raw, Ember) {
    deprecate('ember-ajax/request is deprecated and will be removed in ember-ajax@2.0.0', false, { id: 'ember-ajax.raw' });
    }, null, 'ember-ajax: unwrap raw ajax response');
define('ember-ajax/services/ajax', ['exports', 'ember', 'ember-ajax/errors', 'ember-ajax/utils/parse-response-headers'], function (exports, Ember, ____errors, parseResponseHeaders) {
    import AjaxService from 'ember-ajax/services/ajax';
    import AjaxService from 'ember-ajax/services/ajax';
    import AjaxService from 'ember-ajax/services/ajax';
        deprecate('ember-ajax/ajax#request calling request with `type` is deprecated and will be removed in ember-ajax@1.0.0. If you want to specify a type pass an object like {type: \'DELETE\'}', false, { id: 'ember-ajax.service.request' });
      }, 'ember-ajax: ' + hash.type + ' to ' + url);
define('ember-ajax/utils/parse-args', ['exports'], function (exports) {
define('ember-ajax/utils/parse-response-headers', ['exports'], function (exports) {
define('ember-ajax', ['ember-ajax/index', 'ember', 'exports'], function(__index__, __Ember__, __exports__) {
```